sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/CreateTablePartitioningValidationSuite.scala

```diff
@@ -42,7 +42,7 @@ class CreateTablePartitioningValidationSuite extends AnalysisTest {
     assert(!plan.resolved)
     assertAnalysisError(plan, Seq(
```

Comment:
Use `assertAnalysisErrorClass` instead of `assertAnalysisError` to don't depend on error messages in tests.

Reply:
I forgot it.